### PR TITLE
[OOE-98] Created UDF to send new events to Azure Boards

### PR DESCRIPTION
**Pull request by ccaspanello**

### CODE REVIEW

File: src/main/java/com/overops/azure/boards/AzureBoardsFunction.java [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+            // Authenticate
+            String credential = Credentials.basic(input.authUser, input.authToken);
+
+            // Create HTTP Client
+            OkHttpClient client = new OkHttpClient.Builder().build();
+            MediaType mediaType = MediaType.parse("application/json-patch+json");
+            RequestBody body = RequestBody.create(mediaType, bodyContent);
+
+            // Send Request
+            URL url = new URL("https://dev.azure.com/" + input.organization + "/" + input.project + "/_apis/wit/workitems/$task?api-version=5.0");
```
Comment (DFieldFL) [inline src/main/java/com/overops/azure/boards/AzureBoardsFunction.java:86]: Can this hostname change?

File: src/main/java/com/overops/azure/boards/view/DescriptionGenerator.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+     * @return an HTML string
+     */
+    public String generate(EventView event) {
+        try {
+            TemplateLoader loader = new ClassPathTemplateLoader(){
+                @Override
+                protected URL getResource(String location) {
+                    return DescriptionGenerator.class.getResource(location);
+                }
+            };
```
Comment (DFieldFL) [inline src/main/java/com/overops/azure/boards/view/DescriptionGenerator.java:35]: Maybe add a comment on why you had to do it this way knowing there was class loading issues.

**Review by DFieldFL — CHANGES_REQUESTED** [review]

File: src/main/java/com/overops/azure/boards/view/DescriptionGenerator.java (hunk 2 above)
Comment (ccaspanello) [reply src/main/java/com/overops/azure/boards/view/DescriptionGenerator.java:35]: Good call!

**Review by DFieldFL — APPROVED** [review]

File: src/main/java/com/overops/azure/boards/AzureBoardsFunction.java (hunk 1 above)
Comment (DFieldFL) [reply src/main/java/com/overops/azure/boards/AzureBoardsFunction.java:86]: SaaS is always using the same hostname.

**Pull request merged**